README.md

```diff
@@ -132,6 +132,46 @@ Visual Studio does not support this feature.
 When debugging in regtest, BTCPay Server will automatically create an configure two wallets. (cashcow and merchant)
 You can trigger payments or mine blocks on the invoice's checkout page.
 
+## 6. Integration Tests without Docker
```

Comment:
this method feels a bit like a hack.. the right way to do this would be by attaching to a Docker container.. pls move to a web site post instead